torch/distributions/fishersnedecor.py

```diff
@@ -0,0 +1,96 @@
+from numbers import Number
+import torch
+import math
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions import Beta, Chi2
+from torch.distributions.gamma import _standard_gamma, Gamma
+from torch.distributions.utils import broadcast_all, _finfo
+
+
+class FisherSnedecor(Distribution):
+    r"""
+    Creates a Fisher-Snedecor distribution parameterized by `df1` and `df2`.
+
+    Example::
+
+        >>> m = StudentT(torch.Tensor([1.0]), torch.Tensor([2.0]))

```

Comment:
Update example (it still says "Student T"

Reply:
My bad. Sorry.